library/spdm_crypt_lib/libspdm_crypt_asym.c

```diff
@@ -757,24 +911,55 @@ bool libspdm_asym_verify(
         if (!result) {
             return false;
         }
-        return libspdm_asym_verify_wrap(context, hash_nid, base_asym_algo,
-                                        param, param_size,
-                                        message_hash, hash_size,
-                                        signature, sig_size);
+        result = false;
+        if (try_big_endian) {
+            result = libspdm_asym_verify_wrap(context, hash_nid, base_asym_algo,
+                                              param, param_size,
+                                              message_hash, hash_size,
+                                              signature, sig_size);
+        }
+        if (result == false && try_little_endian) {
```

Comment:
`if (result == false && try_little_endian)`
->
`if (!result && try_little_endian)`

and in subsequent instances.

Reply:
Agree.